model/src/application/interaction/application_command/data/mod.rs

```diff
@@ -26,50 +34,173 @@ pub struct CommandData {
 
 /// Data received when a user fills in a command option.
 ///
-/// Note: user, channel, role, and mentionable option types will be returned as
-/// a [`String`] option here.
-///
 /// Refer to [the discord docs] for more information.
 ///
 /// [the discord docs]: https://discord.com/developers/docs/interactions/application-commands#interaction-applicationcommandinteractiondataoption
```

Comment:
links seems outdated
```suggestion
/// [the discord docs]: https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-interaction-data-option-structure
```